tests/unit/test_notifications.py

```diff
@@ -54,13 +64,14 @@ def _get_call_args(self, target):
         return args
 
     @ddt.data(
-        (1, 'course1', 'some-location', workgroup1, 'Activity1'),
-        (2, 'course2', 'other-location', workgroup1, 'Activity2'),
-        (54, 'course3', 'yet-another-location', workgroup2, 'NotAnActivity'),
+        (1, 'course1', 'some-location', 'stage-location', workgroup1, 'Activity1'),
+        (2, 'course2', 'other-location', 'other-stage-location', workgroup1, 'Activity2'),
+        (54, 'course3', 'yet-another-location', 'yet-another-stage-location', workgroup2, 'NotAnActivity'),
     )
     @ddt.unpack
-    def test_file_upload_success_scenario(self, user_id, course_id, location, workgroup, name):
-        block = ActivityNotificationsGuineaPig(user_id, course_id, location, workgroup, name)
+    def test_file_upload_success_scenario(self, user_id, course_id, location, stage_location, workgroup, name):
+        activity = ActivityNotificationsGuineaPig(user_id, course_id, location, workgroup, name)
```

Comment:
@ziafazal hint: activity location (`location`) is irrelevant here, so you might want to replace it with `'irrelevant'` and remove the parameter from the function (as opposed to suppressing pylint "too-many-arguments" error)